### ISSUE TITLE
Can't you take into account 3D data?

### ISSUE THREAD

**Issue by UnixJunkie** [ComputBiophys/ProtRA#1]
Hello,

If you are given a 3D structure solved experimentally (e.g. in the PDB).
Can't you use that to improve your predictions?
With a structure, we will know which AAs are buried and which
have a chance to be solvent-exposed or exposed to the lipid layer.
Very interesting work!
Thanks a lot for making it open-source!

Regards,
Francois.

**Comment by KKKangKaiKK** [comment]
Thanks for your issue. Our predictor is designed to predict RLA, RSA and RBSA from sequences. If you have a 3D structure, you can:

1. Search for its existence in [MemProtMD databse](http://memprotmd.bioch.ox.ac.uk), which is also the training set of ProtRAP. Or you can execute your own molecular dynamics simulations following their protocol.
2. Use [OPM server](https://opm.phar.umich.edu) to calculate the orientation of the protein in the lipid bilayer.

In summary, if you already have a 3D structure, there are several ways to get more accurate results.

**Comment by UnixJunkie** [comment]
Thanks, I did not know about those.
You know your topic.
